src/spawns.rs

```diff
@@ -149,7 +146,7 @@ pub(crate) fn spawn_actor(
                             }
                             resp_s.send(r).unwrap();
                             continue;
-                        }
+                        }*/
```

Comment:
Can this section be removed or is it going to make a comeback

Reply:
This won't make a comeback. Issue #79 will ultimately put _something_ back here in its place.